README.md

```diff
@@ -80,6 +80,19 @@ make -j4
 ```
 STK can then be run from the build directory with `bin/supertuxkart`
 
+### Recompiling
+
+To recompile the latest code without redownloading the entire source, first run the ```svn up``` command inside the 'stk-assets' directory, then run the following commands inside the 'stk-code' directory:
+
+```git pull
+make clean
```

Comment:
Make clean shouldn't be required, also there is no makefile at the root so that wouldn't work?

Reply:
Ah, yes that makes sense. I must be getting that part confused with the 0ad rebuild process. I'll rebuild again after the next commit and make sure that make clean is definitely not required.

Reply:
The commands worked perfectly without make clean.